docs/sphinx/about/whats-new.txt

```diff
@@ -96,6 +102,11 @@ Other general improvements include:
 * fixes for the detection of csv pattern blocks by `FilePatternBlock`
 * bioformats_package.jar now includes bio-formats-tools as a dependency so
   ImageConverter, ImageFaker and ImageInfo classes are included in the bundle
+* initial work on
+  `Reader discoverability <https://github.com/openmicroscopy/design/issues/42>`_ now allows
+  optional/external Readers to be annotated into the Readers class and
```

Comment:
maybe something along the lines `initial work on Reader discoverability now allows to annotate readers in the `readers.txt` configuration file using key/value pairs`?
I will have a go at the external reader use case in a few weeks so that could be described separately for the next milestone.